lib/fork.js

```diff
@@ -19,8 +19,16 @@ module.exports = function (args) {
 	var ps = childProcess.fork(babel, args, options);
 
 	var promise = new Promise(function (resolve, reject) {
+		// define backup results, when there are no tests defined
+		var testResults = {
+			stats: {
+				passCount: 0
```

Comment:
I think I will revert this one, because it seems unrelated to this PR. I think this question deserves its own space.